mlos_bench/mlos_bench/config/environments/apps/fio/fio.jsonc

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "FIO Composite Env",
+    "class": "mlos_bench.environments.composite_env.CompositeEnv",
+
+    "config": {
+        "children": [
+            {
+                "name": "Upload FIO job file",
+                "class": "mlos_bench.environments.local.LocalFileShareEnv",
+                "config": {
+                    "required_args": [
+                        "experiment_id",
+                        "trial_id"
+                    ],
+                    "upload": [
+                        {
+                            "from": "environments/apps/fio/scripts/remote/",
+                            "to": "$experiment_id/$trial_id/scripts"
+                        }
+                    ]
+                }
+            },
+            {
+                "name": "Run FIO remotely",
+                "class": "mlos_bench.environments.remote.RemoteEnv",
+                "config": {
+                    "required_args": [
+                        "vmName",
+                        "experiment_id",
+                        "trial_id",
+                        "mountPoint"
+                    ],
+                    "setup": [
+                        "sudo apt-get update",
+                        "sudo apt-get -y install fio"
+                    ],
+                    "run": [
+                        "mkdir -p /tmp/mlos_bench/fio/",
+                        "sudo mkdir -p $mountPoint/$experiment_id/$trial_id/output/fio/",
+                        "sudo fio --output-format=json $mountPoint/$experiment_id/$trial_id/scripts/latency-profile.fio > /tmp/mlos_bench/fio/latency-profile.out.json",
```

Comment:
The location fio runs from is often important (e.g., on the VM's data or ephemeral disk instead of the OS disk).
Can we provide a parameter to help control for that?

Reply:
done